google-cloud-pubsub/samples/pubsub_list_schema_revisions.rb

```diff
@@ -20,9 +20,12 @@ def list_schema_revisions schema_id:
 
   pubsub = Google::Cloud::Pubsub.new
 
-  schema = pubsub.schema schema_id
+  schemas = pubsub.schemas
 
-  response = schema.list_revisions
+  view = Google::Cloud::PubSub::V1::SchemaView.const_get "FULL"
```

Comment:
Better is simply:

```ruby
view = Google::Cloud::PubSub::V1::SchemaView::FULL
```

And at that point, I don't think you need the separate `view` local variable. Just put `Google::Cloud::PubSub::V1::SchemaView::FULL` inline in the parameter below.